<ui version="4.0">
 <class>ffmpegTranscoding</class>
 <widget class="QMainWindow" name="ffmpegTranscoding" >
<property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ffmpegTranscoding</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>551</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="openFloder_btn">
       <property name="text">
        <string>选择文件夹</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="Path_text"/>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="cmd_text">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>551</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="copy_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>551</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>复制</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ffmpegTranscoding.qrc"/>
 </resources>
 <connections/>
</ui>